### PR TITLE
[rb]: Standardise driver logging output

### PR DESCRIPTION
**Pull request by luke-hill**
Ensure all log formats (Including browser ones), are in a consistent format

**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description
Alter logging to conform to other standard log formats

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [ ] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by luke-hill** [comment]
Failure seems unrelated

```
 1) Selenium::WebDriver::DevTools network interception changes requests
     Failure/Error: expect(driver.find_elements(tag_name: 'img').map(&:size).uniq).to eq([Dimension.new(640, 480)])
     
       expected: [#<struct Selenium::WebDriver::Dimension width=640, height=480>]
            got: [#<struct Selenium::WebDriver::Dimension width=0, height=0>, #<struct Selenium::WebDriver::Dimension width=640, height=480>]
     
       (compared using ==)
```

Not really sure what happened here, guessing there is a secondary blank 0x0 image somewhere on the page it's finding?

**Comment by titusfortner** [comment]
This isn't a backward compatible change and the code isn't marked as `@api private`, even if maybe it should be. What are you trying to do and how does this change make it better?

File: rb/lib/selenium/webdriver/common/log_entry.rb [hunk 1]
```diff
@@ -22,22 +22,22 @@ module WebDriver
     class LogEntry
       attr_reader :level, :timestamp, :message
 
-      def initialize(level, timestamp, message)
-        @level = level
+      def initialize(timestamp, level, message)
```
Comment (p0deje) [inline rb/lib/selenium/webdriver/common/log_entry.rb]: Is there any value to change the order here? Even though nobody should be initializing this class directly, I don't believe it's worth breaking backwards compatibility without a good reason.

**Review by p0deje — COMMENTED** [review]
I don't quite follow what do we standard against. Do you want the browser logs to look like Ruby's `Logger` class?

File: rb/lib/selenium/webdriver/common/log_entry.rb (hunk 1 above)
Comment (luke-hill) [reply rb/lib/selenium/webdriver/common/log_entry.rb]: I'll undo this change, but it was only used in the entire codebase in one location, which was api private.

**Comment by luke-hill** [comment]
Is this RtM now @p0deje (The ruby failure seems to find a 0x0 reference).

**Comment by p0deje** [comment]
@luke-hill Could you please reply to https://github.com/SeleniumHQ/selenium/pull/9850#pullrequestreview-758637516?

File: rb/lib/selenium/webdriver/common/log_entry.rb (hunk 1 above)
Comment (luke-hill) [reply rb/lib/selenium/webdriver/common/log_entry.rb]: Sorry yes. The other logger formats in a different way, and the timestamp is always first in most loggers. Here it wasn't.

**Pull request merged**